fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -1397,4 +1401,36 @@ public void readFields(DataInput in) throws IOException {
             throw new IOException("error happens when parsing create routine load stmt: " + origStmt, e);
         }
     }
+
+    abstract public void modifyProperties(AlterRoutineLoadStmt stmt) throws DdlException;
```

Comment:
The job properties maybe could be modified in here instead of the subclass  `KafkaRoutineLoad`.

Reply:
Maybe we could do this when we got second derived class from RoutineLoadJob